ingest-app/docs/api.md

```diff
@@ -738,28 +782,242 @@ Example output:
     xmlns:gss="http://www.isotc211.org/2005/gss"
     xmlns:gts="http://www.isotc211.org/2005/gts"
     xmlns:srv="http://www.isotc211.org/2005/srv"
-    xmlns:swe="http://schemas.opengis.net/sweCommon/2.0/"
     xmlns:xlink="http://www.w3.org/1999/xlink"
-    xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
Same comment as above. Is it really necessary to put the full ISO metadata in here?

Reply:
I don't think so, I would be fine with cutting out much of this and replacing it with '…'